R/TealAppDriver.R

```diff
@@ -166,12 +166,15 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
     #' @description
     #' Get the active shiny name space bound with a custom `element` name.
     #'
-    #' @param element `character(1)` custom element name.
+    #' @param element (`character(1)`) custom element name.
+    #' @param hash (`logical(1)`) whether to prepend the name with `#` sign.
     #'
     #' @return (`string`) The active shiny name space of the component bound with the input `element`.
-    active_module_element = function(element) {
+    active_module_element = function(element, hash = TRUE) {
```

Comment:
Can you provide use case where id is needed without a `#` prefix?

Reply:
@gogonzo yes, it is for the download button of the table in `table_with_settings`

https://github.com/insightsengineering/teal.widgets/pull/240/commits/aba68837a8cded32799a1301c1a67ef1c3d1cd55#diff-62f05f4a2946a28d82c9a50384d362f9c47e6af156367fb8a9bc485bf1bb2f32R155

Reply:
it's the `app$get_value()` that doesn't take `#`

Reply:
Yes, but teal.widgets don't use tealAppDriver so I think it isn't necessary

Reply:
Yeah, I've never experience a need to remove hash anywhere else in any other test